docs/permission/overview.md

```diff
@@ -0,0 +1,41 @@
+---
+id: overview
+title: Permission Framework Overview
+description: A high level overview of the Backstage permission framework
+---
+
+[The previous section](../auth/index.md) covered the various _authentication_ methods of Backstage, but Backstage can also _authorize_ specific data, APIs, or interface actions - meaning that Backstage has the ability to enforce rules about what type of access is allowed for a given user of a system.
+
+By default, Backstage endpoints are not protected, and all actions are available to anyone. However, configuring which users can access which resources and actions is a common need for many organizations. The permission framework allows integrators to achieve this through the use of granular permissioning for those resources and actions.
+
+The permission framework was designed with a few key properties in mind:
+
+- Flexibility: the framework allows integrators to configure many different authorization methods. This could include implementations like role-based access control (RBAC), attribute-based access control (ABAC), bespoke logic expressed in code, or integrations with external authorization providers.
+
+- Usability: the permission framework allows integrators to focus on configuring what they care about (the permission policy) by providing all of its moving parts out of the box. It also allows plugin authors to integrate support for permissions in their plugins without having to make any changes in Backstage core.
+
+## How does it work?
```

Comment:
Thoughts on adding a "Concepts" section to nail down important terminology used throughout the docs? See the [plugin analytics docs](https://backstage.io/docs/plugins/analytics#concepts) for example

Reply:
Sounds like a good idea to me!

Reply:
I really like this idea, and I would take it a step further and say we should include a separate page for this (kind of like the appendix from the PRFC, but focused more on concrete definitions of important concepts of the framework). I can take this on as a follow up.